tutorials/nservicebus-sagas/2-timeouts/Snippets/Core_7/BuyersRemorseTimeoutRequest/BuyersRemorseTimeoutRequest.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Core_7.BuyersRemorseTimeoutRequest
+{
+    using NServiceBus;
+    using NServiceBus.Logging;
+    using System;
+    using System.Threading.Tasks;
+
+    class BuyersRemorsePolicy : Saga<BuyersRemorseState>
```

Comment:
Yay! Here is how I like to see it. Unfortunately not inside a region, so no snippet ;-)